case_studies.md

```diff
@@ -0,0 +1,30 @@
+# Turing Way - case studies for reproducible research
+
+Reproducible research is work that can be independently verified. 
+In practice, it means sharing the data and code that were used to generate published results - yet this is often easier said than done.
+The Turing Way is a guide to reproducible data science that will support students and academics as they develop their code, with the aim of helping them produce work that will be regarded as gold-standard examples of trustworthy and reusable research. 
+It is openly developed and any and all questions, comments and recommendations are welcome at our GitHub repository: https://github.com/alan-turing-institute/the-turing-way 
```

Comment:
Do Google forms let you set up hyperlinks? If so, could you connect the link to "GitHub repository".

Reply:
Just tried to fix this with html and have to say that unfortunately the answer is no unless I'm overseeing something super obvious :disappointed:

Reply:
No worries; the links aren't that long so let's leave them in. Thanks for investigating!